quarkus/admin/build.gradle.kts

```diff
@@ -108,17 +102,10 @@ val quarkusBuild = tasks.named<QuarkusBuild>("quarkusBuild")
 val distTar =
   tasks.named<Tar>("distTar") {
     dependsOn(quarkusBuild)
-    // Trigger resolution (and build) of the run-script artifact
-    inputs.files(runScript)
     compression = Compression.GZIP
   }
 
-val distZip =
-  tasks.named<Zip>("distZip") {
-    dependsOn(quarkusBuild)
-    // Trigger resolution (and build) of the run-script artifact
-    inputs.files(runScript)
-  }
+val distZip = tasks.named<Zip>("distZip") { dependsOn(quarkusBuild) }
```

Comment:
Do we still need this?

Reply:
We shouldn't need it anymore, neither does the tarball. Let me remove it.

Reply:
Removed in a new commit